apache-cassandra-mixin/dashboards/cassandra-keyspaces.libsonnet

```diff
@@ -708,13 +708,13 @@ local keyspaceReadLatencyPanel(matcher) = {
       'sum(cassandra_keyspace_readlatency_seconds{' + matcher + ', quantile="0.95"} >= 0) by (keyspace)',
       datasource=promDatasource,
       legendFormat='{{ keyspace }} - p95',
-      format='heatmap',
+      format='time_series',
     ),
     prometheus.target(
       'sum(cassandra_keyspace_readlatency_seconds{' + matcher + ', quantile="0.99"} >= 0) by (keyspace)',
       datasource=promDatasource,
       legendFormat='{{ keyspace }} - p99',
-      format='heatmap',
+      format='time_series',
```

Comment:
@Dasomeone to be fair, it has not. I changed this because I was seeing this error for this panel on the original dashboard with no current data being pulled.
<img width="461" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/12396806/0bd34fa2-5ca3-4981-9e23-31e0e112596c">
Looking at the fields for the sister "writelatency" panel, I noticed that the format was set to "time_series" which matched the panel "type" field. For some reason this one had this inconsistency (it's supposed to be a timeseries and not a heatmap).